### DISCUSSION TITLE
Issues with launcher updating.

### DISCUSSION THREAD

**Discussion by joshricker** [SKCraft/Launcher#497] [I need help!]
Tring to use the launchers ability to update its self and I see and click the update launcher button. It then tells me to restart the launcher. But once I close and restart the launcher its still the old version. I can see its created a folder called "launcher" and inside it is the updated jar, just renamed to something like 1674874852910.jar. If I drag that out it works, but I'm sure this is not how its meant to work. I'm using the fancy launcher.

**Comment by hedgehog1029** [comment]
it sounds like you may be running without the bootstrapper (https://github.com/SKCraft/Launcher/wiki/Configuring-Self-Update)?

The launcher should probably not show the self-update button when running in standalone mode, since it... doesn't work, as you've found. On Windows if the launcher tried to overwrite the JAR it's running from, it throws a fit about the file being in use, hence why it needs the bootstrapper to launch it in the first place.
